tests/Unit/Console/Commands/CacheAnnualStatisticsTest.php

```diff
@@ -94,12 +100,14 @@
         ]);
 
     Transaction::factory()
-        ->multiPayment()
+        ->multiPayment(
+            $addresses,
+            [BigNumber::new(10 * 1e18), BigNumber::new(1 * 1e18)]
+        )
         ->withReceipt()
         ->create([
+            'amount' => BigNumber::new(10 * 1e18)->plus(1 * 1e18),
             'timestamp' => $timestamp,
```

Comment:
Waiting for confirmation that the amount on the multi-payment transaction matches the sum of all payments, which is required for calculating the stats